packages/react-popover/src/components/PopoverTrigger/usePopoverTrigger.ts

```diff
@@ -21,23 +31,20 @@ export const usePopoverTrigger_unstable = (props: PopoverTriggerProps): PopoverT
   const trapFocus = usePopoverContext_unstable(context => context.trapFocus);
   const { triggerAttributes } = useModalAttributes();
 
-  const onContextMenu = useEventCallback((e: React.MouseEvent<HTMLElement>) => {
+  const onContextMenu = (e: React.MouseEvent<HTMLElement>) => {
     if (openOnContext) {
       e.preventDefault();
       setOpen(e, true);
     }
+  };
 
-    child.props?.onContextMenu?.(e);
```

Comment:
I moved callbacks merging directly to `applyTriggerPropsToChildren()` as in other components.